jsonnet/kube-prometheus/kube-prometheus-thanos-sidecar.libsonnet

```diff
@@ -5,10 +5,10 @@ local servicePort = k.core.v1.service.mixin.spec.portsType;
 {
   _config+:: {
     versions+:: {
-      thanos: 'v0.5.0',
+      thanos: 'v0.7.0',
     },
     imageRepos+:: {
-      thanos: 'improbable/thanos',
+      thanos: quay.io/thanos/thanos',
```

Comment:
There is a missing single quote, if my eyes are correct.

Reply:
@peterbueschel nicely spotted, do you want to open a PR to fix this?

Reply:
Oh, sorry for that miss. I will open a PR to fix